Need sys/wait.h for WIFEXITED and WEXITSTATUS
Use gtar(1) not tar(1) due to dynamic compression support

Index: src/libexpr/primops/fetchGit.cc
--- src/libexpr/primops/fetchGit.cc.orig
+++ src/libexpr/primops/fetchGit.cc
@@ -6,6 +6,7 @@
 #include "hash.hh"
 
 #include <sys/time.h>
+#include <sys/wait.h>
 
 #include <regex>
 
@@ -173,7 +174,7 @@ GitInfo exportGit(ref<Store> store, const std::string 
     Path tmpDir = createTempDir();
     AutoDelete delTmpDir(tmpDir, true);
 
-    runProgram("tar", true, { "x", "-C", tmpDir }, tar);
+    runProgram("gtar", true, { "x", "-C", tmpDir }, tar);
 
     gitInfo.storePath = store->addToStore(name, tmpDir);
 
